src/Server.ts

```diff
@@ -14,8 +17,13 @@ const { app }=this;
 app.get('/health-check',( req, res, next ) => {
 res.send("I am fine");
 });
+app.use(notFoundHandler);
+app.use(errorHandler);
 return this;
 }
+initBodyParser() {
+    this.app.use(bodyParser.json({ type: 'application/*+json' }))
```

Comment:
its better to use most recommended way to use body-parser with Express.
from the doc
Express route-specific
This example demonstrates adding body parsers specifically to the routes that need them. In general, this is the most recommended way to use body-parser with Express.

Reply:
done